frontend/src/libs/components/loader/loader.tsx

```diff
@@ -0,0 +1,32 @@
+import { DefaultLoaderColor } from './enum/default-color.enum.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  children: React.ReactNode;
+  color?: string;
+  isLoading: boolean;
+};
+
+const Loader: React.FC<Properties> = ({
+  children,
+  color = DefaultLoaderColor.BORDER,
+  isLoading,
+}) => {
+  if (!isLoading) {
+    return <>{children}</>;
+  }
+
+  return (
+    <div className={styles.container}>
+      <div
+        className={styles.loader}
```

Comment:
Better provide default color in styles and if we'd need to provide custom styles - then it would be provided by className from optional prop
```suggestion
        className={getValidClassNames(styles.loader, className)}
```